lib/utils/reify-output.js

```diff
@@ -73,10 +78,17 @@ const reifyOutput = arb => {
 // report.  if we get here, we know it's not quiet or json.
 const printAuditReport = arb => {
   const reporter = npm.command !== 'audit' ? 'install' : 'detail'
-  const res = auditReport(arb.auditReport, {
-    reporter,
-    ...npm.flatOptions
-  })
+  let res
+  if (arb.auditReport) {
```

Comment:
Note: we need to check for `arb.auditReport.error` to see if there was an error loading the audit report.  Otherwise it'll look like an audit ran and found no vulns, when really it's just that the audit service is unreachable or something.

Reply:
added an issue so we don't forget: https://github.com/npm/statusboard/issues/125